.semaphore/semaphore.yml

```diff
@@ -0,0 +1,26 @@
+version: v1.0
+name: AppSignal sql_lexer
+agent:
+  machine:
+    type: e1-standard-2
+  containers:
+    - name: main
+      image: registry.semaphoreci.com/rust:1.51
+blocks:
+  - name: 'Default'
+    task:
+      jobs:
+        - name: Formatter and linter
+          commands:
+            - checkout
+            - rustup component add rustfmt clippy
+            - cache restore clippy-${SEMAPHORE_GIT_BRANCH}-${LOCK_CHECKSUM}
```

Comment:
The `LOCK_CHECKSUM` env var seems to be unset. From the build output:

```
cache restore clippy-${SEMAPHORE_GIT_BRANCH}-${LOCK_CHECKSUM}
MISS: clippy-add-semaphore-ci-
```

Reply:
Yes, I forgot to export it with `Cargo.lock` checksum, but it's ignored here. We don't need cache for this build.